docs/schema_mapping.py

```diff
@@ -107,6 +107,20 @@
         'JSON Schema Validation',
         'All the literal values in the enum are included in the definition.'
     ],
+    [
+        'SecretStr',
+        'string',
+        '{"writeOnly": true}',
+        'Pydantic standard "format" extension',
```

Comment:
Sorry, forgot to mention. After removing `format`, this no longer applies: `'Pydantic standard "format" extension',`.

Reply:
What should it be instead? JSON Schema Validation or nothing?

Reply:
:man_facepalming: I didn't say it above.

It should be: `JSON Schema Validation`